### PR TITLE
created get classroom/classrooms and update api

### PR DESCRIPTION
**Pull request by jako2889**

### CODE REVIEW

File: private/api/classrooms/get-classroom.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?php
+require_once('../db-connection/root-connection.php');
+try{
+
+    if(isset($_GET['classroom_id'])){
+        //GET ID
+        $classroom_id = $_GET['classroom_id'];
+        //Prepare SQL query
+        $q = $db->prepare('SELECT * FROM classrooms WHERE id = :classroom_id');
```
Comment (magn3506) [inline private/api/classrooms/get-classroom.php:9]: classroom_id in SQL . Should it be classroomID ? Lets discuss :)

File: private/api/classrooms/update-classroom.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<?php
+require_once('../db-connection/root-connection.php');
+try{
```
Comment (magn3506) [inline private/api/classrooms/update-classroom.php:3]: we need to implement validation of input so i matches DB-types. Maybe create a TODO:  ? :)

**Review by magn3506 — CHANGES_REQUESTED** [review]

File: private/api/classrooms/get-classroom.php (hunk 1 above)
Comment (jako2889) [reply private/api/classrooms/get-classroom.php:9]: Skulle vi ikke følge syntaksen vi havde lavet? <3

File: private/api/classrooms/update-classroom.php (hunk 2 above)
Comment (jako2889) [reply private/api/classrooms/update-classroom.php:3]: will do

**Review by magn3506 — APPROVED** [review]

**Pull request merged**